src/screens/Registration/Registration.js

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from "react";
 import PropTypes from "prop-types";
-import { Text, ScrollView, KeyboardAvoidingView } from "react-native";
+import { Text, ScrollView, KeyboardAvoidingView, View } from "react-native";
```

Comment:
Why do you import View here if it isn't used?

Reply:
fixed